unitq/README.md

```diff
@@ -0,0 +1,43 @@
+# unitQ
+
+## Overview
+
+unitQ is your consolidated, searchable platform for user feedback. Our AI technology extracts data-driven insights from what users are saying to help you increase product quality.
+
+Our solution monitors millions of quality issues daily - for category-leading brands like Chime, Quizlet, Pandora, and Strava.
```

Comment:
We try to avoid sales related material for the integrations page so I think we could remove this

Reply:
Done!